src/services/tx/tx-sender/__tests__/ts-sender.test.ts

```diff
@@ -30,6 +30,30 @@ const setupFetchStub = (data: any) => (_url: string) => {
 }
 import type { EIP1193Provider, OnboardAPI, WalletState, AppState } from '@web3-onboard/core'
 import { hexZeroPad } from 'ethers/lib/utils'
+import { generatePreValidatedSignature } from '@safe-global/safe-core-sdk/dist/src/utils/signatures'
+
+const createSafeTx = (data = '0x'): SafeTransaction => {
```

Comment:
There is a function `createMockSafeTransaction`
I'd suggest using that and extending it by additional mock capabilities if needed.